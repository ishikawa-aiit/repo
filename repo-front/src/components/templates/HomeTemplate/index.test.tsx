import { HomeTemplate } from '.'

import { renderWithQueryClient } from '.jest'

const title = 'サンプルタイトル'
const description = 'サンプルディスクリプション'

describe('snapshots', () => {
  it('最新のスナップショットに更新されている', () => {
    const { container } = renderWithQueryClient(<HomeTemplate title={title} description={description} />)

    expect(container).toMatchSnapshot()
  })
})
