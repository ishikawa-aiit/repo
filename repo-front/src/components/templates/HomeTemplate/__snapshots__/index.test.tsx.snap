// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots 最新のスナップショットに更新されている 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  min-height: 100vh;
  margin-top: 16px;
}

.emotion-1 {
  width: 100%;
  height: 100%;
  overflow: visible;
}

.emotion-2 {
  position: relative;
}

.emotion-2::before {
  display: block;
  content: "";
  width: 384px;
  height: 617px;
  position: absolute;
  top: -84px;
  left: 0px;
  background: url("/images/pages/home/bg_left-obj_01.svg") no-repeat 0 0;
  z-index: -1;
}

.emotion-3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  width: 1200px;
  margin: 20px auto 0;
}

.emotion-4 {
  width: 500px;
}

.emotion-5 {
  margin-bottom: 10px;
}

.emotion-7 {
  font-size: xl;
  font-weight: bold;
  line-height: 1.5;
}

.emotion-8 {
  font-size: sm;
  margin-top: 2px;
}

.emotion-9 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-appearance: none;
  -moz-appearance: none;
  -ms-appearance: none;
  appearance: none;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  position: relative;
  white-space: nowrap;
  vertical-align: middle;
  outline: 2px solid transparent;
  outline-offset: 2px;
}

.emotion-10 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  -webkit-margin-end: 0.5rem;
  margin-inline-end: 0.5rem;
}

.emotion-12 {
  width: 620px;
}

.emotion-13 {
  position: relative;
}

.emotion-13::before {
  display: block;
  content: "";
  width: 688px;
  height: 477px;
  position: absolute;
  top: -52px;
  right: 0px;
  background: url("/images/pages/home/bg_right-obj_01.svg") no-repeat 0 0;
  z-index: -1;
}

.emotion-14 {
  width: 1264px;
  margin: 0 auto;
}

.emotion-15 {
  font-size: 30px;
  margin-top: 120px;
  margin-bottom: 50px;
  padding-bottom: 6px;
  text-align: center;
  position: relative;
}

.emotion-15::after {
  display: block;
  content: "";
  width: 4rem;
  height: 4px;
  background-color: primary.500;
  border-radius: 9999px;
  position: absolute;
  left: 50%;
  bottom: 0px;
  -webkit-transform: translateX(-50%);
  -moz-transform: translateX(-50%);
  -ms-transform: translateX(-50%);
  transform: translateX(-50%);
}

.emotion-16 {
  display: grid;
  grid-template-columns: repeat(3, minmax(0, 1fr));
  margin-bottom: 40px;
}

.emotion-17 {
  padding: 8px;
}

.emotion-18 {
  margin-bottom: 6px;
}

.emotion-20 {
  padding: 8px;
  border-left: 1px;
  border-color: neutral.200;
}

.emotion-26 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  color: neutral.600;
  width: 100%;
  margin-top: auto;
  font-size: sm;
  padding-top: 3px;
  padding-bottom: 3px;
  border-top: 1px;
  border-top-color: neutral.200;
}

<div>
  <div
    class="emotion-0"
    data-cy="HomeLayout"
    data-testid="HomeLayout"
  >
    <main
      class="emotion-1"
      data-cy="Main"
      data-testid="Main"
    >
      <div
        class="emotion-2"
        data-cy="HomeTitle"
        data-testid="HomeTitle"
      >
        <div
          class="emotion-3"
        >
          <div
            class="emotion-4"
          >
            <div
              class="emotion-5"
            >
              <h1
                class="chakra-heading emotion-5"
              >
                <span
                  style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
                >
                  <span
                    style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                  >
                    <img
                      alt=""
                      aria-hidden="true"
                      src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27400%27%20height=%27181%27/%3e"
                      style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                    />
                  </span>
                  <img
                    alt="週報作成支援ツール Repo"
                    data-nimg="intrinsic"
                    decoding="async"
                    src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                    style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                  />
                  <noscript />
                </span>
              </h1>
              <p
                class="chakra-text emotion-7"
              >
                AIIT（東京都立産業技術大学院大学）学生のPBL週報作成を効率化。manaba上での週報作成を最短時間で完了できます。また、manaba上では参照しづらい過去の週報も簡単に参照できます。
              </p>
              <p
                class="chakra-text emotion-8"
              >
                ※ Google Chrome以外のブラウザの場合、表示が崩れる場合があります。
              </p>
            </div>
            <button
              class="chakra-button emotion-9"
              data-cy="LinkToCreationPage"
              data-testid="RedirectButton"
              type="button"
            >
              <span
                class="chakra-button__icon emotion-10"
              >
                <svg
                  aria-hidden="true"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 488 512"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M488 261.8C488 403.3 391.1 504 248 504 110.8 504 0 393.2 0 256S110.8 8 248 8c66.8 0 123 24.5 166.3 64.9l-67.5 64.9C258.5 52.6 94.3 116.6 94.3 256c0 86.5 69.1 156.6 153.7 156.6 98.2 0 135-70.4 140.8-106.9H248v-85.3h236.1c2.3 12.7 3.9 24.9 3.9 41.4z"
                  />
                </svg>
              </span>
              <span
                class="chakra-text emotion-11"
              >
                ログインして週報作成
              </span>
            </button>
          </div>
          <div
            class="emotion-12"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27620%27%20height=%27496%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt=""
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </span>
          </div>
        </div>
      </div>
      <div
        class="emotion-13"
        data-cy="HomeFeature"
        data-testid="HomeFeature"
      >
        <div
          class="emotion-14"
        >
          <h2
            class="chakra-heading emotion-15"
          >
            Repoの特徴
          </h2>
          <div
            class="emotion-16"
          >
            <div
              class="emotion-17"
            >
              <h3
                class="chakra-heading emotion-18"
              >
                簡単に過去の週報を見返せる！
              </h3>
              <p
                class="chakra-text emotion-11"
              >
                 
                週報一覧から参照したい週報を容易にストレスなく閲覧することができます。
              </p>
            </div>
            <div
              class="emotion-20"
            >
              <h3
                class="chakra-heading emotion-18"
              >
                週報を素早く記入できる！
              </h3>
              <p
                class="chakra-text emotion-11"
              >
                 
                週報作成時に登録したフォーマットを反映させることで、入力時間を大幅に短縮できます。
              </p>
            </div>
            <div
              class="emotion-20"
            >
              <h3
                class="chakra-heading emotion-18"
              >
                週報の記入を忘れない！
              </h3>
              <p
                class="chakra-text emotion-11"
              >
                 
                週報提出期限が近づくと、リマインダーを受け取ることができます。
              </p>
            </div>
          </div>
        </div>
      </div>
    </main>
    <footer
      class="emotion-26"
      data-cy="Footer"
      data-testid="Footer"
    >
      © Advanced Institute of Industrial Technology, Oikawa Project Team 2022
    </footer>
  </div>
</div>
`;
